### DISCUSSION TITLE
Nominate @ForestEckhardt for the role of Maintainer

### DISCUSSION THREAD

**Discussion by thitch97** [paketo-buildpacks/feedback#175] [Web Servers Team]
I'd like to nominate @ForestEckhardt for the role of Web Servers Maintainer. He has made a number of contributions in recent months to move the project forward including enabling [default config generation in httpd](https://github.com/paketo-buildpacks/httpd/commit/8ae672eb466f1bc40d9c062589ca5f597d541fe0) & [support for frontend server configuration](https://github.com/paketo-community/web-servers/commit/5e76a65484f2cd9ab917b8deec65c602c6a61f4c). He has shown an interest in the advancement and overall health of the Web Servers CNBs and I believe he would be a positive addition to the sub-team.

https://github.com/orgs/paketo-buildpacks/teams/web-servers-maintainers
https://github.com/orgs/paketo-buildpacks/teams/steering-committee 👍/👎?

**Comment by arjun024** [comment]
👍

**Comment by ryanmoran** [comment]
Congrats @ForestEckhardt! You are now a Web Servers Maintainer.
